src/styles/Blocks/card.scss

```diff
@@ -0,0 +1,81 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 40px;
+}
+
+.card__title {
+  @include fonts($size: 12px, $line: 18px);
+  color: $secondary-color;
+  margin: 0 16px 4px;
+}
+
+.card__description {
+  @include fonts($weight: 400);
+  color: #616070;
+  margin: 0 16px 16px;
+}
+
+.card__ratings {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin: 0 16px 24px;
+}
+
+.card__reviews {
+  @include fonts($weight: 400);
+  color: $secondary-color;
+  margin: 0;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 16px 16px;
+}
+
+.card__price-title {
+  font-weight: 400;
+  color: #616070;
+  margin: 0;
+}
+
+.card__amount {
+  @include fonts($size: 16px, $weight: 700, $line: 18px);
+  margin: 0;
+}
+
+.card__button {
+  @include fonts($size: 14px, $weight: 700);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-decoration: none;
+  background-color: $primary-color;
+  margin: 0 16px 16px;
+  border: 1px solid transparent;
+  border-radius: 5px;
+  color: #fff;
+  text-transform: uppercase;
+  transition: 0.3s;
+  box-sizing: border-box;
+  width: 166px;
+  height: 40px;
```

Comment:
you have box-sizing in the main so no need here. Also Ihor's comment for my button was like, "use paddings not height" so maybe you can also do it :)